test/parallel/test-vm-context.js

```diff
@@ -92,16 +93,17 @@ assert.strictEqual(script.runInContext(ctx), false);
 
 // Error on the first line of a module should
 // have the correct line and column number
+let assertErrStack = null;
 assert.throws(() => {
-  vm.runInContext(' throw new Error()', context, {
+  vm.runInContext('throw new Error()', context, {
     filename: 'expected-filename.js',
     lineOffset: 32,
     columnOffset: 123
   });
 }, (err) => {
-  return /^ \^/m.test(err.stack) &&
-         /expected-filename\.js:33:131/.test(err.stack);
-}, 'Expected appearance of proper offset in Error stack');
+  assertErrStack = err.stack;
+  return /expected-filename\.js:33:130/.test(err.stack);
+}, `expected appearance of proper offset in Error stack: ${assertErrStack}`);
```

Comment:
This test lost two parts that were tested here before. Testing for `^` was definitely intended and should be further tested for. I am also relatively sure the space in front of the throw was intended as well to test for the correct column number.

It would probably be best to just remove the error message overall.